File: admin_frontend/lib/models/organization.dart
class Organization {
  final int id;
  final int? userId;
  final String name;
  final String? description;
  final String status;
  final String? website;
  final String? contactEmail;
  final String? country;
  final String? type;
  final DateTime createdAt;

  Organization({
    required this.id,
    this.userId,
    required this.name,
    this.description,
    required this.status,
    this.website,
    this.contactEmail,
    this.country,
    this.type,
    required this.createdAt,
  });

  factory Organization.fromJson(Map<String, dynamic> json) {
    return Organization(
      id: json['id'],
      userId: json['user_id'],
      name: json['name'],
      description: json['description'],
      status: json['status'],
      website: json['website'],
      contactEmail: json['contact_email'],
      country: json['country'],
      type: json['type'],
      createdAt: json['created_at'] != null 
          ? DateTime.parse(json['created_at']) 
          : DateTime.now(),
    );
  }
}

File: admin_backend/app/schemas/organization.py
from pydantic import BaseModel, EmailStr
from typing import Optional, List, Any
from datetime import datetime

class OrganizationBase(BaseModel):
    name: str
    description: Optional[str] = None
    website: Optional[str] = None
    contact_email: Optional[EmailStr] = None
    country: Optional[str] = None
    type: Optional[str] = None
    status: str = "pending"

class OrganizationCreate(OrganizationBase):
    user_id: int
    verification_documents: Optional[List[str]] = None

class OrganizationUpdate(BaseModel):
    name: Optional[str] = None
    description: Optional[str] = None
    website: Optional[str] = None
    contact_email: Optional[EmailStr] = None
    status: Optional[str] = None
    verification_documents: Optional[List[str]] = None

class Organization(OrganizationBase):
    id: int
    user_id: int
    verification_documents: Optional[Any] = None
    created_at: datetime
    updated_at: Optional[datetime] = None

    class Config:
        from_attributes = True
